### ISSUE TITLE
Text node whitespace behaviour

### ISSUE THREAD

**Issue by wkrueger** [riazXrazor/html-to-jsx#3]
React (or JSX) intentionally "deviates" from HTML in which, when a text node encounters a line break, HTML will add a white space, but react wont.

Thats probably why sometimes prettier adds `{' '}` while reformatting.

https://github.com/facebook/react/issues/1643

It would be great if this extension also took that in consideration when converting.

Great work, thx

**Comment by wkrueger** [comment]
ok, this is on the shoulders of https://github.com/reactjs/react-magic

**Comment by riazXrazor** [comment]
@wkrueger yup thats correct i just wrapped it for to be used in vscode :), since i had to frequntly has to convert html to jsx
